pipelines/common/src/main/java/org/openmrs/analytics/ParquetUtil.java

```diff
@@ -274,7 +276,7 @@ GenericRecord convertToAvro(Resource resource) {
     // can fail for some R4 resource types (and they should be excluded). Once this is done we
     // should remove the requirement for specifying resourceTypes when running the pipeline.
     org.hl7.fhir.dstu3.model.Resource r3Resource =
-        org.hl7.fhir.convertors.VersionConvertor_30_40.convertResource(resource, true);
+        new VersionConvertor_30_40(new BaseAdvisor_30_40()).convertResource(resource);
```

Comment:
Do we need to set the `failFast` variable?
 https://hapifhir.io/hapi-fhir/apidocs/hapi-fhir-converter/org/hl7/fhir/convertors/advisors/impl/BaseAdvisor_30_40.html

Reply:
I think the default is `true`; do you want it to be `false`? The documentation does not say much but looking at the code it doesn't seem to do much anyways.
This whole conversion logic is going away very soon in PR #354.

Reply:
If it is going away soon, its OK to have the default